src/modules/attributes.ts

```diff
@@ -1,17 +1,28 @@
 import {VNode, VNodeData} from '../vnode';
 import {Module} from './module';
 
-const NamespaceURIs = {
-  "xlink": "http://www.w3.org/1999/xlink"
-};
-
 const booleanAttrs = ["allowfullscreen", "async", "autofocus", "autoplay", "checked", "compact", "controls", "declare",
                 "default", "defaultchecked", "defaultmuted", "defaultselected", "defer", "disabled", "draggable",
                 "enabled", "formnovalidate", "hidden", "indeterminate", "inert", "ismap", "itemscope", "loop", "multiple",
                 "muted", "nohref", "noresize", "noshade", "novalidate", "nowrap", "open", "pauseonexit", "readonly",
                 "required", "reversed", "scoped", "seamless", "selected", "sortable", "spellcheck", "translate",
                 "truespeed", "typemustmatch", "visible"];
 
+const xlinkNS = 'http://www.w3.org/1999/xlink';
+const xmlNS = 'http://www.w3.org/XML/1998/namespace';
+
+const namespaces = Object.create(null);
+
+namespaces['xlink:href'] = xlinkNS;
```

Comment:
@diervo we are missing `xlink:show`. this is the major difference. This PR makes the list very restrictive, which I'm ok with, but we will have to maintain it if more attributes are added at some point.